man/journalctl.xml

```diff
@@ -18,7 +18,7 @@
 
   <refnamediv>
     <refname>journalctl</refname>
-    <refpurpose>Query the systemd journal</refpurpose>
+    <refpurpose>Print log entries from the the systemd journal</refpurpose>
```

Comment:
maybe say "Print log entries from and control the systemd journal"?

After all, a good chunk of what it does is send control commands, i.e. not just print log entries